tekton/publish.yaml

```diff
@@ -95,7 +95,7 @@ spec:
 
       # Combine Distroless with a Windows base image, used for the entrypoint image.
       COMBINED_BASE_IMAGE=$(go run ./vendor/github.com/tektoncd/plumbing/cmd/combine/main.go \
-        gcr.io/distroless/base:debug-nonroot \
+        ghcr.io/distroless/busybox \
```

Comment:
Also, I think the same sentiment applies to ALL of these, since they are all tested with `static:nonroot` in CI, not just `entrypoint` (which you specifically called out).

Reply:
Ok, so thinking through this has made me realize this use of `busybox` is wholly distinct from the shell-images cases, so I am going to forget about these, and you should go nuts switching things to OG `static:nonroot`, and then we can revisit this for NG `static` as part of a later convo.